core/src/main/java/org/mskcc/cbio/oncokb/util/GenomeNexusUtils.java

```diff
@@ -215,18 +227,43 @@ private static String getIsoform(Gene gene, ReferenceGenome referenceGenome) {
         }
     }
 
-    private static VariantAnnotation getVariantAnnotation(GNVariantAnnotationType type, String query, ReferenceGenome referenceGenome) throws ApiException {
-        VariantAnnotation variantAnnotation = null;
-        if (query != null && type != null) {
+    public static String chromosomeNormalizer(String chromosome) {
+        return chromosome.trim().replace("chr", "").replace("23", "X").replace("24", "Y");
+    }
+
+    private static GenomicLocation convertGenomicLocation(String genomicLocation) {
```

Comment:
Was this method introduced for performance reasons? NotationConverter also has a method to parse genomic location

Reply:
Issue seems to be util method can't use the NotationConverter class.